src/main/java/eggventory/storage/Storage.java

```diff
@@ -31,20 +34,26 @@ public Storage(String filePath) {
     public StockList load() {
         StockList savedList = new StockList();
 
-        if (Files.notExists(Paths.get(filePath))) {
+        if (Files.notExists(Paths.get(stockFilePath)) || Files.notExists(Paths.get(stockTypesFilePath))) {
             try {
                 Files.createDirectory(Paths.get("data/"));
             } catch (IOException e) {
                 System.out.println("Unknown IO error when creating 'data/' folder.");
             }
         }
-        File f = new File(filePath);
-
+        File f = new File(stockTypesFilePath);
+        File f2 = new File(stockFilePath);
         try {
-            Scanner s = new Scanner(f); //Create a Scanner using the File as the source
+            Scanner s = new Scanner(f); //Create a Scanner to read saved_stocktypes file
+            Scanner s2 = new Scanner(f2); //Create a Scanner using the File as the source
 
             while (s.hasNext()) {
                 String itemRaw = s.nextLine();
+                AddStockTypeCommand cmd1 = new AddStockTypeCommand(CommandType.ADD, itemRaw);
+                cmd1.execute(savedList);
+            }
+            while (s2.hasNext()) {
+                String itemRaw = s2.nextLine();
                 String[] item = itemRaw.split(",", 0);
```

Comment:
clean and simple. very nice. maybe in a future update can give s and s2 more descriptive names, but functionality is clear nonetheless.